lab_2_tokenize_by_bpe/main.py

```diff
@@ -36,6 +54,16 @@ def count_tokens_pairs(
     :param word_frequencies: dictionary in the form of <preprocessed word: number of occurrences>
     :return: dictionary in the form of <token pair: number of occurrences>
     """
+    if not isinstance(word_frequencies, dict):
+        return None
+    pair_dictionary = {}
+    for word, frequency in word_frequencies.items():
+        for i in range(len(word)-1):
+            if (word[i], word[i+1]) in pair_dictionary:
```

Comment:
simplify - set 0 if there is not the key in the dict and do += always

Reply:
fixed